opendbc/car/docs_definitions.py

```diff
@@ -312,8 +312,8 @@ def init(self, CP: CarParams, all_footnotes=None):
     # hardware column
     hardware_col = "None"
     if self.car_parts.parts:
-      model_years = self.model + (' ' + self.years if self.years else '')
-      buy_link = f'<a href="https://comma.ai/shop/comma-3x.html?make={self.make}&model={model_years}">Buy Here</a>'
+      car_info = f"{self.make} {self.model}{' ' + self.years if self.years else ''}".replace(' ', '%20')
```

Comment:
```suggestion
      car_info = f"{self.make} {self.model}{' ' + self.years if self.years else ''}"
```

sure this is needed?

Reply:
why nott use use car_info.name?